source/MaterialXView/Handlers/ObjGeometryHandler.cpp

```diff
@@ -331,13 +331,21 @@ void ObjGeometryHandler::readData()
         _normalData.push_back(norm[vertexIndex + 2]);
 
         // Fake some tangent, bitangent data
-        _tangentData[0].push_back(norm[vertexIndex]);
-        _tangentData[0].push_back(norm[vertexIndex + 1]);
         _tangentData[0].push_back(norm[vertexIndex + 2]);
```

Comment:
Minor change to generate additional tangent + bitangent sets now that it's supported in the spec.